src/main/java/mate/academy/internetshop/service/impl/BucketServiceImpl.java

```diff
@@ -23,8 +24,9 @@ public Bucket create(Bucket bucket) {
 
     @Override
     public Bucket get(Long id) throws NoSuchElementException {
-        if (bucketDao.get(id).isPresent()) {
```

Comment:
return bucketDao.get(id).orElseThrow(() -> new NoSuchElementException("Can't find bucket with id: " + id));

Reply:
Ok. Will fix with the next push.